.github/workflows/lint.yml

```diff
@@ -17,21 +17,21 @@ jobs:
       -
         name: Check out repository code
         uses: actions/checkout@v3
+      -
+        uses: technote-space/get-diff-action@v6.1.2
+        with:
+            PATTERNS: |
+             **/**.go
+             go.mod
+             go.sum
+             .github/**
+             Makefile
       -
         name: 🐿 Setup Golang
         uses: actions/setup-go@v4
         with:
           go-version: ${{env.GO_VERSION}}
-      -
-        uses: technote-space/get-diff-action@v6.1.2
-        if: env.GIT_DIFF
-        with:
-          PATTERNS: |
-            **/**.go
-            go.mod
-            go.sum
-            .github/**
-            Makefile
```

Comment:
Note: `if: env.GIT_DIFF` was breaking it because this environment variable is set by the git diff action itself.

Additionally, I moved to be before setting up golang so that we don't redundantly set it up

Reply:
LOL thats hilarious, woops

Sorry all, my bad